libs/wingsdk/src/cloud/website.ts

```diff
@@ -0,0 +1,95 @@
+import { isAbsolute, resolve } from "path";
```

Comment:
Noticed another issue when working with the website resource on aws:
Even after we manually invalidated the CloudFront cache - `terraform apply` didn't publish index.html correctly.

Reply:
I think it takes time and depends on the invalidation method, I'll try implementing somehow the invalidation and we'll see if it persists

Reply:
Ok, thanks, but please note that we (ShaiA and myself) waited for the cache invalidation to complete, then viewed index.html and it was still outdated.